src/providers/ad/ad_gpo.c

```diff
@@ -864,19 +864,20 @@ ad_gpo_filter_gpos_by_dacl(TALLOC_CTX *mem_ctx,
 
         access_allowed = false;
         candidate_gpo = candidate_gpos[i];
-        sd = candidate_gpo->gpo_sd;
-        dacl = candidate_gpo->gpo_sd->dacl;
 
         DEBUG(SSSDBG_TRACE_ALL, "examining dacl candidate_gpo_guid:%s\n",
                                 candidate_gpo->gpo_guid);
 
         /* gpo_func_version must be set to version 2 */
         if (candidate_gpo->gpo_func_version != 2) {
             DEBUG(SSSDBG_TRACE_ALL,
-                  "GPO not applicable to target per security filtering\n");
+                  AD_AT_FUNC_VERSION" of this GPO is not 2. Skipping.\n");
             continue;
```

Comment:
I think we can have both sentences in commit message.  Removed sentence looks good enough for me.
There is just missing "Skipping". But feel free to extend it with your your sentence.